src/types/venues.ts

```diff
@@ -46,7 +46,7 @@ export interface Venue_v2_Base {
   host: {
     icon: string;
   };
-  owners?: string[];
+  owners: string[];
```

Comment:
Similar concerns/caveats as other places in this PR where I have mentioned making this optional, and whether we can guarantee that it will be set for all venues (including currently existing data within firestore)

Reply:
False type, it's not going to be a problem.

Reply:
Sounds good